src/services/dhcp.md

```diff
@@ -20,6 +23,109 @@ The Internet Systems Consortium's implementation of a DHCP server is good enough
 
 ## Configuration 
 
-TODO
+```
+# option definitions common to all supported networks...
+option domain-name "cslabs.clarkson.edu";
+option domain-name-servers 1.1.1.1, 1.0.0.1;
 
+default-lease-time 600;
+max-lease-time 7200;
 
+# The ddns-updates-style parameter controls whether or not the server will
+# attempt to do a DNS update when a lease is confirmed. We default to the
+# behavior of the version 2 packages ('none', since DHCP v2 didn't
+# have support for DDNS.)
+ddns-update-style none;
+
+# If this DHCP server is the official DHCP server for the local
+# network, the authoritative directive should be uncommented.
+authoritative;
+
+# Use this to send dhcp log messages to a different log file (you also
+# have to hack syslog.conf to complete the redirection).
+#log-facility local7;
```

Comment:
```suggestion
# log-facility local7;
```

Reply:
not sure if the space should be there or not but wanted to make sure before the merge